js/index.js

```diff
@@ -0,0 +1,16 @@
+const menuBtn = document.getElementById('menu-btn');
+const closeBtn = document.getElementById('close-btn');
+const menu = document.getElementById('menu');
+menuBtn.addEventListener('click', () => {
+  // console.log('Clicked');
```

Comment:
- [ ] [optional] It's best practice to keep your code clean. So I recommend removing unnecessary commented lines of code